source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java

```diff
@@ -216,7 +217,7 @@ public static void setBackgroundImage(RenderedAdaptiveCard renderedCard,
 
     public static void setSelectAction(RenderedAdaptiveCard renderedCard, BaseActionElement selectAction, View view, ICardActionHandler cardActionHandler, RenderArgs renderArgs)
     {
-        if (selectAction != null)
+        if (selectAction != null && selectAction.GetIsEnabled())
         {
             view.setFocusable(true);
```

Comment:
nit: should it be focusable? will Android VO be able to narrate the disabled element?

Reply:
No, TalkBack will not narrate the element. 

From the spec (#4670):

> Images or other elements with a disabled selectAction look and behave as if they had no selectAction.